web/docs/interfaces/Fields.md

```diff
@@ -194,6 +210,126 @@ In conversation tracking, B to A bytes counter per conversation
 
 ___
 
+### IcmpType
+
+• `Optional` **IcmpType**: `number`
+
+ICMP type
+
+___
+
+### IcmpCode
+
+• `Optional` **IcmpCode**: `number`
+
+ICMP code
+
+___
+
+### PktDropLatestState
+
+• `Optional` **PktDropLatestState**: `string`
+
+Pkt TCP state for drops
+
+___
+
+### PktDropLatestDropCause
+
+• `Optional` **PktDropLatestDropCause**: `string`
+
+Pkt cause for drops
+
+___
+
+### PktDropLatestFlags
+
+• `Optional` **PktDropLatestFlags**: `number`
+
+Pkt TCP flags for drops
+
+___
+
+### PktDropPackets
+
+• `Optional` **PktDropPackets**: `number`
+
+Number of packets dropped by the kernel in this flow
```

Comment:
```suggestion
Number of packets dropped by the kernel in a specific flow
```

Reply:
I'm not sure about that: all those fields are related to a given flow, hence the `this`. If I read `in a specific flow` then I would wonder: which flow are we talking about? Whereas `this` is contextual. I think I can just remove the `in this flow` part, as it isn't ambiguous anyway.

Reply:
If the `in this flow` part can be removed that may be best because when I read `this` I was also wondering which flow is `this`? I can see how it may be more clear in context though, like you say.